frontend/src/scenes/plugins/source/pluginSourceLogic.tsx

```diff
@@ -115,23 +119,30 @@ export const pluginSourceLogic = kea<pluginSourceLogicType>([
                     : ['plugin.json', 'index.ts'],
         ],
     }),
-
-    listeners(({ props, values }) => ({
+    beforeUnload(({ actions, values }) => ({
```

Comment:
For some reason this fires twice:
![2022-10-05 14 18 05](https://user-images.githubusercontent.com/4550621/194060963-348d4019-cf32-4510-93c3-7f557f99102b.gif)

Might be because first the URL is changed by the browser on back click, and then in `urlToAction` calls `hidePluginLogs()` which results in `actionToUrl` pushing the same URL for `hidePluginLogs`?